src/config/env.ts

```diff
@@ -61,4 +61,9 @@ export interface EnvVars {
 	JIRA_LINK_TRACKING_ID?: string,
 	PROXY?: string,
 	LAUNCHDARKLY_KEY?: string;
+
+	// Micros Lifecycle Env Vars
+	SNS_NOTIFICATION_LIFECYCLE_QUEUE_URL?:string;
+	SNS_NOTIFICATION_LIFECYCLE_QUEUE_NAME?:string;
+	SNS_NOTIFICATION_LIFECYCLE_QUEUE_REGION?:string;
```

Comment:
How does it work? Would Micros spin up a queue per node?

Reply:
A queue per ec2 instance as per the documentation.